views/welcome.html

```diff
@@ -10,6 +10,7 @@ <h3>To get started, select a task below:</h3>
       <a class="list-group-item list-group-item-action" id="list-standby-list" data-toggle="list" href="#list-standby" role="tab" aria-controls="standby">Deploy Standby</a>
       <a class="list-group-item list-group-item-action" id="list-follower-list" data-toggle="list" href="#list-follower" role="tab" aria-controls="followers">Deploy Followers</a>
       <a class="list-group-item list-group-item-action" id="list-integration-list" data-toggle="list" href="#list-integration" role="tab" aria-controls="integration">Secure Apps</a>
+      <a class="list-group-item list-group-item-action" id="list-create-azure-host-policy-list" data-toggle="list" href="#list-create-azure-host-policy" role="tab" aria-controls="integration">Azure Authenticator</a>
```

Comment:
The rest of the titles our of actions. I think we should change this one to "Generate Azure Authenticator host policy"